Base.rte/Constants.lua

```diff
@@ -10,6 +10,7 @@ rte.StartingFundsScale = 1.0;
 rte.DiggersRate = 0.4;
 rte.MetabaseArea = "MetabaseServiceArea";
 rte.PxTravelledPerFrame = GetPPM() * TimerMan.DeltaTimeSecs;
+rte.PathFindingDefaultDigStrength = 35; -- The default dig strength used for pathfinding. Matches the value defined in the source code.
```

Comment:
Why add this? What is it useful for?

Reply:
It's used in bunker breach - a couple things get the number of steps in the path as a measure of closeness (since scene man shortest distance doesn't account for walls), and I figure it's best to use a consistent default pathfinding strength for this.
I guess the better thing would have been to expose the cpp version but I didn't think about that at the time lol

Reply:
Either-or is good to me. Obviously exposing from C++ is a bit more effort, but I think it's okay either way.